src/node.h

```diff
@@ -504,6 +504,9 @@ struct IsolateSettings {
 // feature during the build step by passing the --disable-shared-readonly-heap
 // flag to the configure script.
 //
+// The snapshot *must* be kept alive during the execution of the environment /
+// isolate that it is derived from.
```

Comment:
```suggestion
// The snapshot *must* be kept alive during the execution of the Isolate
// that derives from it.
```

or

```suggestion
// The snapshot *must* be kept alive during the execution of the Isolate
// that was created using it.
```

(this header refers to `Isolate`/`Environment` as capitalized names to distinguish them from the regular English-language words; `Environment` instances do not outlive their `Isolate` instances; and the `Isolate` derives from the snapshot in this scenario, not the other way around, since this is about the case in which the snapshot is being consumed, not when it is being built)